internal/cmd/init/init.go

```diff
@@ -30,6 +30,10 @@ var gradleGroovySetup string
 //go:embed instructions/gradlekotlin
 var gradleKotlinSetup string
 
+const (
+	GradleMultiProjectWarningMsg = "For multi-project builds, you should setup cifuzz in the subprojects containing the fuzz tests."
+)
```

Comment:
I think this message can be longer to be more helpful.

E.g.

```suggestion
	GradleMultiProjectWarningMsg = `For multi-project builds, you should run cifuzz init in each of the sub-projects
that contain fuzz tests and configure their build configuration individually.`
)
```

```

Reply:
I am not sure what this should clarify? In Gradle it is clear that you define the needed dependencies in the subproject,,, what else do you mean with "build configuration"?

Reply:
the specific build configuration files (e.g. `build.gradle`, etc.)
Just a suggestion

Reply:
this should all be there already and it is not part of what cifuzz deal with... so if it is ok for you, I just would keep it like this